src/hooks/accounts/useGetAccounts.tsx

```diff
@@ -11,37 +9,27 @@ export const basiliskAddressPrefix = 10041;
  * Used to fetch either all accounts, or accounts matching the given
  * set of parameters e.g. 'isActive'.
  */
-export const useGetAccounts = () => useCallback(async (
-    persistedActiveAccountId: string | undefined,
-    isActive: boolean | undefined
-) => {
+export const useGetAccounts = () =>
+  useCallback(async () => {
     // TODO: use `config.appName`
     // ensure we're connected to the polkadot.js extension
     await web3Enable('basilisk-ui');
 
     // get all the accounts from the polkadot.js extension
     const accounts = (await web3Accounts())
-        // transform the returned accounts into the required entity format
-        .map(account => {
-            const address = encodeAddress(decodeAddress(account.address), basiliskAddressPrefix)
-            return {
-                id: address,
-                name: account.meta.name,
-                isActive: false,
-            };
-        })
-        // mark the active account based on the current persisted account
-        .map(account => ({
-            ...account,
-            isActive: persistedActiveAccountId == account.id,
-        }))
-
-    // if we're filtering for an active account, return just the single active account
-    if (isActive) {
-        const account = find(accounts, { isActive: isActive })
-        return account;
-    }   
+      // transform the returned accounts into the required entity format
+      .map((account) => {
+        const address = encodeAddress(
+          decodeAddress(account.address),
+          basiliskAddressPrefix
+        );
+        return {
+          id: address,
```

Comment:
We should also include the genesisHash or whatever else comes from the extension to identify what chain the account is configured as.

Reply:
This is what comes in for my testing account. I'm putting in source for now, let's add anything else when it's needed
```
{
    "address": "5DwWSMW7kqBUU4Eo6s5dDvpJvKpXRgVZc24DUqqVXHH6UmFt",
    "meta": {
        "genesisHash": "",
        "name": "Vac Slavik",
        "source": "polkadot-js"
    },
    "type": "sr25519"
}
```